ewatercycle/util.py

```diff
@@ -86,6 +86,16 @@ def lat_lon_boundingbox_to_variable_indices(model, variable, latMin, latMax, lon
     return np.array(output)
 
 
+def convert_timearray_to_datetime(timearray: xr.DataArray) -> datetime:
```

Comment:
Function is no longer called in any of the code. As we are using `LisfloodForcing.start_time` instead of looking in NetCDF files. 

Can this method and its test be removed?

Reply:
It is removed.